change-camera-controller/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,11 @@
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```

Comment:
```suggestion
    <style name="AppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
```
The sample will look weird if `DayNight` is used since the colors for the Dark theme have not been set up yet.

Reply:
I believe quite a few samples using the `DayNight` theme. We might want to change it everywhere in that case.

Reply:
There is an issue in progress to address this :)